includes/User.php

```diff
@@ -35,7 +35,7 @@
  * Int Serialized record version.
  * @ingroup Constants
  */
-define( 'MW_USER_VERSION', 8 );
+define( 'MW_USER_VERSION', 9 );
```

Comment:
Basically a cache buster. Since some people now have their memcache data completely incorrect without an expiration, this will force a load from the database for all users.